tests/options.test.ts

```diff
@@ -9,6 +9,8 @@ defineTest("maximum margin", {data: "test", margin: 10000000}, `"margin" must be
 
 defineTest("invalid type", {data: "test", type: "foo"}, `"type" must be one of [png, jpeg]`);
 
```

Comment:
I suggest adding a test with an invalid property of `Itheme`, for example:

```suggestion

defineTest("invalid theme property", {data: "test", theme: {foreground: "black", fooBar: "wrong"}}, ...);
```

Reply:
won't do as we switch the option to a filepath

Reply:
This is an API test, not a CLI one.